src/main/java/com/epam/novostroinyi/core/waiter/SelenideWaiter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.novostroinyi.core.waiter;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import com.codeborne.selenide.Condition;
+import com.epam.novostroinyi.core.ui.element.UiElement;
+import com.epam.novostroinyi.core.ui.element.UiElementsCollection;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import org.openqa.selenium.WebElement;
+
+public class SelenideWaiter implements ConditionalWaiter {
+
```

Comment:
introduce implicit wait and several explicit waits in configs. Do not hardcode the timeouts.